Client/List.ts

```diff
@@ -46,14 +46,20 @@ export abstract class List<
 			page: number,
 			size: number,
 			sort?: string,
+			providerCode?: model.ProviderCode,
+			includeCount?: boolean,
 			request?: Record<string, any>
 		) => Promise<model.ErrorResponse | { list: R[]; totalCount: number } | R[]>,
 		request?: Record<string, any>,
 		chosenPage?: number,
 		chosenSize?: number,
-		chosenSort?: string
+		chosenSort?: string,
+		chosenProvider?: model.ProviderCode,
+		chosenIncludeCount?: boolean
 	) {
 		const sort = chosenSort
+		const provider = chosenProvider
+		const includeCount = chosenIncludeCount
```

Comment:
the provider and includeCount don't need to be and shouldn't be in this very generic method. they only apply to a couple of endpoints

they should only be in the endpoint specific methods, above ^